openapi/api2pdf/Package.md

```diff
@@ -0,0 +1,19 @@
+Connects to [Api2Pdf REST API](https://www.api2pdf.com/) from Ballerina
+
+## Package overview
+The `ballerinax/api2pdf` is a [Ballerina](https://ballerina.io/) connector for Api2Pdf.
+This package provides the capability to access Api2Pdf API.
+
+### Compatibility
+|                               | Version                        |
+|-------------------------------|--------------------------------|
+| Ballerina Language            | Ballerina Swan Lake Beta3      |
+| Api2Pdf REST API              | V2.0.0                         | 
+
+## Report issues
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina connector repository](https://github.com/ballerina-platform/ballerinax-openapi-connectors)
```

Comment:
This has to point the extended-library repo

Reply:
Did the suggested changes.